src/Nest/XPack/MachineLearning/Job/Results/CategoryDefinition.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject]
+	public class CategoryDefinition
+	{
+		[JsonProperty("job_id")]
```

Comment:
This PR is full of awesome xmldocs but this one is left out 😿

Reply:
Done.